### PR TITLE
Integration with SpartakCore

### PR DESCRIPTION
**Pull request by Spartak1997**

### CODE REVIEW

File: src/main/java/com/github/technus/tectech/loader/recipe/RecipeLoader.java [hunk 1]
```diff
@@ -110,8 +111,12 @@ public void run() {
 
         if (Loader.isModLoaded(Reference.DREAMCRAFT)) {
             new DreamCraftRecipeLoader().run();//init recipes for GTNH version
-        } else {
+        } else 
+        if (Loader.isModLoaded(Reference.SPARTAKCORE)) {
```
Comment (Technus) [inline src/main/java/com/github/technus/tectech/loader/recipe/RecipeLoader.java]: no new line in  else if

File: src/main/java/com/github/technus/tectech/loader/recipe/RecipeLoader.java [hunk 2]
```diff
@@ -110,8 +111,12 @@ public void run() {
...
+        	new SpartakCoreRecipeLoader().run();//init recipes for SpartakCore version
+        }else {
```
Comment (Technus) [inline src/main/java/com/github/technus/tectech/loader/recipe/RecipeLoader.java]: } else {

File: src/main/java/com/github/technus/tectech/Reference.java [hunk 3]
```diff
@@ -12,6 +12,7 @@
     public static final String THAUMCRAFT = "Thaumcraft";
     public static final String DREAMCRAFT = "dreamcraft";
     public static final String GTPLUSPLUS = "miscutils";
+    public static final String SPARTAKCORE = "spartakcore";
```
Comment (Technus) [inline src/main/java/com/github/technus/tectech/Reference.java:15]: does it even require other recipes that default GT5u recipes from bloody loader?

**Review by Technus — CHANGES_REQUESTED** [review]
Minor changes + explanation required

File: src/main/java/com/github/technus/tectech/Reference.java (hunk 3 above)
Comment (Spartak1997) [reply src/main/java/com/github/technus/tectech/Reference.java:15]: No, I  will write recipes in my core. Maybe you know I use a fork from GTNH GT and it's a reason for writing new core for deeper integration without bloody recipes. I hope you understand me.

**Review by Technus — APPROVED** [review]

**Pull request merged**